[2023-05-02T01:06:15.437+0000] {taskinstance.py:1084} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: get_todays_gamesv2.check_game_today_task manual__2023-05-02T01:06:08.690760+00:00 [queued]>
[2023-05-02T01:06:15.457+0000] {taskinstance.py:1084} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: get_todays_gamesv2.check_game_today_task manual__2023-05-02T01:06:08.690760+00:00 [queued]>
[2023-05-02T01:06:15.457+0000] {taskinstance.py:1282} INFO - 
--------------------------------------------------------------------------------
[2023-05-02T01:06:15.457+0000] {taskinstance.py:1283} INFO - Starting attempt 1 of 1
[2023-05-02T01:06:15.458+0000] {taskinstance.py:1284} INFO - 
--------------------------------------------------------------------------------
[2023-05-02T01:06:15.482+0000] {taskinstance.py:1303} INFO - Executing <Task(PythonOperator): check_game_today_task> on 2023-05-02 01:06:08.690760+00:00
[2023-05-02T01:06:15.489+0000] {standard_task_runner.py:55} INFO - Started process 33278 to run task
[2023-05-02T01:06:15.496+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'get_todays_gamesv2', 'check_game_today_task', 'manual__2023-05-02T01:06:08.690760+00:00', '--job-id', '780', '--raw', '--subdir', 'DAGS_FOLDER/todays_gamesv2.py', '--cfg-path', '/tmp/tmpe6jbbblp']
[2023-05-02T01:06:15.497+0000] {standard_task_runner.py:83} INFO - Job 780: Subtask check_game_today_task
[2023-05-02T01:06:15.638+0000] {task_command.py:388} INFO - Running <TaskInstance: get_todays_gamesv2.check_game_today_task manual__2023-05-02T01:06:08.690760+00:00 [running]> on host f789e77bc70a
[2023-05-02T01:06:15.815+0000] {taskinstance.py:1511} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=get_todays_gamesv2
AIRFLOW_CTX_TASK_ID=check_game_today_task
AIRFLOW_CTX_EXECUTION_DATE=2023-05-02T01:06:08.690760+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-05-02T01:06:08.690760+00:00
[2023-05-02T01:06:15.820+0000] {logging_mixin.py:137} INFO - 2023-05-02
[2023-05-02T01:06:15.852+0000] {taskinstance.py:1775} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 175, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 192, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/todays_gamesv2.py", line 47, in _check_game_today
    game_date = games[0]
KeyError: 0
[2023-05-02T01:06:15.867+0000] {taskinstance.py:1326} INFO - Marking task as FAILED. dag_id=get_todays_gamesv2, task_id=check_game_today_task, execution_date=20230502T010608, start_date=20230502T010615, end_date=20230502T010615
[2023-05-02T01:06:15.887+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 780 for task check_game_today_task (0; 33278)
[2023-05-02T01:06:15.917+0000] {local_task_job.py:212} INFO - Task exited with return code 1
[2023-05-02T01:06:15.960+0000] {taskinstance.py:2585} INFO - 0 downstream tasks scheduled from follow-on schedule check
